test/datapackage.js

```diff
@@ -180,4 +180,36 @@ describe('Datapackage', () => {
       assert(_.isEqual(data, expectedData), 'Wrong data loaded')
     })
   })
+
+  describe('README', () => {
+    const testDatapackage = 'http://bit.do/datapackage-json'
```

Comment:
This link currently is pointing to the `data/dp3-inline/datapackage.json` github repo from keitaro. It would be good if we upload this on FD domain.

I think it is best if the users have some copy-paste examples to try out, that's why I opted for outside URL for the README tests.